### PR TITLE
Docker integration

### PR DESCRIPTION
**Pull request by mykolaKhlopyk**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        condition: service_healthy
+  mongodb:
+    image: mongo
+    container_name: mongodb
+    ports:
+      - "27017:27017"
+    volumes:
+      - data:/data
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=rootuser
```
Comment (BohdanRysakov) [inline docker-compose.yml:19]: Optional for now, but make sure not to show credentials in future

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  telegram-bot:
+    build: .
+    container_name: telegram-bot
+    ports:
+      - "8080:8080"
+    depends_on:
+      mongodb:
+        condition: service_healthy
+  mongodb:
+    image: mongo
```
Comment (BohdanRysakov) [inline docker-compose.yml]:
```suggestion
    image: mongo : 6.0
```
Good practice to specify image's tag. Don't trust latest version

File: run-docker-compose.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./gradlew assemble
+
+docker-compose up --build --force-recreate
```
Comment (BohdanRysakov) [inline run-docker-compose.sh]: This file or unnecessary to commit or run gradle tasks in DockerFile.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,4 @@
+FROM openjdk:17
+EXPOSE 8080
+COPY build/libs/EnglishStudyTelegramBot-*.jar telegram-bot.jar
+ENTRYPOINT ["java", "-jar", "telegram-bot.jar"]
```
Comment (BohdanRysakov) [inline Dockerfile]:
```suggestion
ENTRYPOINT ["java", "-jar", "telegram-bot.jar"]

```

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    container_name: mongodb
+    ports:
+      - "27017:27017"
+    volumes:
+      - data:/data
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=rootuser
+      - MONGO_INITDB_ROOT_PASSWORD=rootpass
```
Comment (BohdanRysakov) [inline docker-compose.yml:20]: Optional for now, but make sure not to show credentials in future

**Review by ajax-shulepa-a — APPROVED** [review]

**Pull request merged**